test/e2e/auth/audit.go

```diff
@@ -65,6 +67,20 @@ var _ = SIGDescribe("Advanced Audit", func() {
 		apiExtensionClient, err := clientset.NewForConfig(config)
 		framework.ExpectNoError(err, "failed to initialize apiExtensionClient")
 
+		if !framework.IsRBACEnabled(f) {
+			framework.Skipf("RBAC not enabled")
+		}
```

Comment:
The other tests can still run w/o RBAC. Just skip the validation of the decision if RBAC is disabled.

Reply:
Done.